src/components/header.svelte

```diff
@@ -1,19 +1,37 @@
 <script lang="ts">
     import { page } from "$app/stores";
+    import { darkModeNavbar } from "$lib/store";
+    import Burger from "./burger.svelte";
     export let items = [];
```

Comment:
we should update navbar items in the layout, why TLIP is duplicated :sweat_smile: